packages/flutter_tools/templates/create/lib/main.dart.tmpl

```diff
@@ -69,6 +69,8 @@ class _MyHomePageState extends State<MyHomePage> {
 
   @override
   Widget build(BuildContext context) {
+    // This method is rerun every time setState is called, for instance
```

Comment:
Did we consider putting this comment above build() ?

I almost read this as "the following method" , because usually comments are above the thing they are commenting on.